PrebidMobile/Objc/PrebidMobileRendering/Networking/Parameters/PBMParameterBuilderService.m

```diff
@@ -139,8 +139,10 @@ + (nonnull PBMORTBBidRequest *)createORTBBidRequestWithTargeting:(nonnull Target
     }
         
     NSValue * const coordObj = targeting.coordinate;
-    if (coordObj) {
-        const CLLocationCoordinate2D coord2d = coordObj.MKCoordinateValue;
+    if (coordObj && CLLocationCoordinate2DIsValid(coordObj.MKCoordinateValue)) {
```

Comment:
Adding `CLLocationCoordinate2DIsValid(coordObj.MKCoordinateValue)` into the if condition can change the current behavior and lead to regression. Can we move 

```const CLLocationCoordinate2D coord2d = coordObj.MKCoordinateValue;```

back to the if's scope?